appendix-basic65.tex

```diff
@@ -264,41 +264,41 @@ \subsection{BASIC 65 Operators}
 For example, the {\bf *} (asterisk) operator performs a multiplication of two number operands. The operator and its operands evaluate to the result of the multiplication.
 
 \begin{screencode}
-A=6
-PRINT A*7
+A = 6
+PRINT A * 7
 \end{screencode}
 
 The {\bf +} (plus) operator has a different meaning depending on the type of the operands. If both operands are numbers, then the operator performs an addition of the numbers. If both operands are strings, then the operator evaluates to a new string that is the concatenation of the operands.
 
 \begin{screencode}
-A=64
-PRINT A+1
+A = 64
+PRINT A + 1
 
-A$="MEGA"
-PRINT A$+"65"
+A$ = "MEGA"
+PRINT A$ + "65"
 \end{screencode}
 
 The {\bf -} (minus) operator accepts either one operand or two operands. Given one number operand on the right-hand side, it evaluates to the negation of that number. Given two number operands, one on either side, it evaluates to the subtraction of the second operand from the first operand.
 
 \begin{screencode}
-A=64
+A = 64
 PRINT -A
 
-PRINT A-16
+PRINT A - 16
 \end{screencode}
 
-The {\bf =} (equals) symbol is used both as an assignment statement and as a relational operator. As an assignment, the {\bf =} symbol is a statement that updates the value of a variable. The left-hand side must be a variable or array element reference, and its type must match the type of the expression on the right-hand side. The assignment is not an operator: it is not part of an expression.
+The equals symbol ({\bf =}) is used both as an assignment statement and as a relational operator. As an assignment, the '{\bf =}' symbol is a statement that updates the value of a variable. The left-hand side must be a variable or array element reference, and its type must match the type of the expression on the right-hand side. The assignment is not an operator: it is not part of an expression.
```

Comment:
Remove the single quotes around the symbol. The typesetting offsets the symbol. I'm generally averse to using single quotes in this way, both for grammatical and typographical reasons. There's not really an English construct based on single quotes other than representing quotes within quotes.